editor/editor_node.cpp

```diff
@@ -2221,16 +2236,26 @@ void EditorNode::_edit_current(bool p_skip_foreign) {
 		int subr_idx = current_res->get_path().find("::");
 		if (subr_idx != -1) {
 			String base_path = current_res->get_path().substr(0, subr_idx);
-			if (FileAccess::exists(base_path + ".import")) {
-				editable_warning = TTR("This resource belongs to a scene that was imported, so it's not editable.\nPlease read the documentation relevant to importing scenes to better understand this workflow.");
+			if (!base_path.is_resource_file()) {
+				if (FileAccess::exists(base_path + ".import")) {
+					if ((get_edited_scene() && get_edited_scene()->get_scene_file_path() == base_path)) {
```

Comment:
Too many parentheses.
```suggestion
					if (get_edited_scene() && get_edited_scene()->get_scene_file_path() == base_path) {
```